### ISSUE TITLE
Can you share the training script on SBD dataset?

### ISSUE THREAD

**Issue by chenxiaoyu523** [holyseven/PSPNet-TF-Reproduce#24]

**Comment by holyseven** [comment]
I lost the best script, and tried this:
``` bash
python ./run_all.py --network 'resnet_v1_101' --visible_gpus '0,3' --weight_decay_mode 1 --weight_decay_rate 0.001 --weight_decay_rate2 0.001 --database 'SBD' --subsets_for_training 'train' --batch_size 8 --train_image_size 480 --train_max_iter 30000 --test_image_size 480 --random_rotate 1 --random_scale 1 --fine_tune_filename './z_pretrained_weights/resnet_v1_101.ckpt'
```
and got 77.72 with single test. With more tuning the learning rate and weight decay rate 1 and 2, the performance could be better.
